plugin.video.pcloud-video-streaming/.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
```

Comment:
Drop this file and also `plugin.video.pcloud-video-streaming/.vscode/.ropeproject/objectdb` from the Pull Request.